QMigrate/plot/trigger.py

```diff
@@ -66,120 +66,88 @@ def trigger_summary(events, starttime, endtime, run, marginal_window,
 
-    if events is not None:
-        for i, event in events.iterrows():
-            lab1 = "Minimum event interval" if i == 0 else ""
-            lab2 = "Marginal window" if i == 0 else ""
-            lab3 = "Triggered event" if i == 0 else ""
-
-            min_dt = event["MinTime"].datetime
-            max_dt = event["MaxTime"].datetime
-            mw_stt = (event["CoaTime"] - marginal_window).datetime
-            mw_end = (event["CoaTime"] + marginal_window).datetime
-            for ax in axes[:2]:
-                ax.axvspan(min_dt, mw_stt, label=lab1, alpha=0.2, color="#F03B20")
-                ax.axvspan(mw_end, max_dt, alpha=0.2, color="#F03B20")
-                ax.axvspan(mw_stt, mw_end, label=lab2, alpha=0.2, color="#3182BD")
-                ax.axvline(event["CoaTime"].datetime, label=lab3, lw=0.01,
-                           alpha=0.4)
-
-    axes[0].text(0.01, 0.925, "Maximum coalescence", ha="left", va="center",
-                 transform=axes[0].transAxes, fontsize=18)
-    axes[0].set_ylabel("Coalescence", fontsize=14)
-    axes[1].text(0.01, 0.925, "Normalised maximum coalescence", ha="left",
-                 va="center", transform=axes[1].transAxes, fontsize=18)
-    axes[1].set_ylabel("Normalised coalescence", fontsize=14)
-    axes[2].text(0.01, 0.925, "Station availability", ha="left", va="center",
-                 transform=axes[2].transAxes, fontsize=18)
-    axes[2].set_ylabel("Available stations", fontsize=14)
-    axes[2].set_xlabel("DateTime", fontsize=14)
+    # --- Plot xy files on map ---
+    _plot_xy_files(xy_files, fig.axes[3])
 
+    # --- Plot event scatter on LUT and windows on coalescence traces ---
     if events is not None:
+        _plot_event_windows(axes[:2], events, marginal_window)
+        _plot_event_scatter(fig, events)
+
+        # Add trigger threshold to the correct coalescence trace
         ax_i = 1 if normalise_coalescence else 0
         axes[ax_i].step(dt, detection_threshold, where="mid", c="g",
-                        label="Detection threshold")
-
-        # Get bounds for cmap
-        vmin, vmax = events["TRIG_COA"].min(), events["TRIG_COA"].max()
-
-        # Plotting the scatter of the earthquake locations
-        x, y, z = events["COA_X"], events["COA_Y"], events["COA_Z"]
-        c = events["TRIG_COA"]
-        sc = axes[3].scatter(x, y, s=50, c=c, cmap="viridis", vmin=vmin,
-                             vmax=vmax)
-        axes[4].scatter(x, z, s=50, c=c, cmap="viridis", vmin=vmin, vmax=vmax)
-        axes[5].scatter(z, y, s=50, c=c, cmap="viridis", vmin=vmin, vmax=vmax)
-
-        # --- Add colourbar ---
-        spec = GridSpec(9, 18).new_subplotspec((2, 7), colspan=1, rowspan=5)
-        cax = fig.add_subplot(spec)
-        cax.set_axis_off()
-        cbar = fig.colorbar(sc, ax=cax, orientation="vertical", fraction=0.4)
-        cbar.ax.set_ylabel("Peak coalescence value", rotation=90, fontsize=14)
+                        label="Trigger threshold")
```

Comment:
Hmm maybe this should stay as "Detection threshold" given that's the parameter name / sort of what it is. I made this change late last night..